README.md

```diff
@@ -1 +1,126 @@
-# map-parking
\ No newline at end of file
+# Park with Access
+
+## Description
+
+Parking with Access App was born out of a collective desire to address the challenges faced by individuals with accessibility requirements when searching for suitable parking spots. Our motivation stems from a genuine commitment to enhancing the mobility and convenience of every user, ensuring that everyone can access parking facilities tailored to their needs.
+
+We built this project not just as a technical exercise but as a response to a real-world issue. Traditional navigation apps often lack comprehensive information about parking facilities, especially those catering to individuals with specific accessibility needs. The Parking Finder App aims to bridge this gap, providing a user-friendly platform that simplifies the process of finding, filtering, and understanding parking options.
```

Comment:
I don't know, feels like we are selling our app a bit too high. Our features right now does not even come close to achieve this ...